### PR TITLE
Added --file-ext feature

### PR DESCRIPTION
**Pull request by ramonck**
Due to command-line does parsing of all files and it´s recursive, gave me a problem of when I have images or whatever other type of file.
I want to filter and only have html extension being parsed by html-minifier, this is the feature I wanted and I created.

Thanks a lot for this project, hope you guys like my contribution.

### CODE REVIEW

**Comment by alexlamsl** [comment]
Thanks for the work - please fix the [style errors](https://travis-ci.org/kangax/html-minifier/jobs/141682983).

**Comment by ramonck** [comment]
I fixed it I think :), please check.

File: cli.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ var mainOptions = {
   customAttrSurround: ['Arrays of regex\'es that allow to support custom attribute surround expressions (e.g. <input {{#if value}}checked="checked"{{/if}}>)', parseJSONRegExpArray],
   customEventAttributes: ['Arrays of regex\'es that allow to support custom event attributes for minifyJS (e.g. ng-click)', parseJSONRegExpArray],
   decodeEntities: 'Use direct Unicode characters whenever possible',
+
```
Comment (alexlamsl) [inline cli.js:110]: Any reason for the extra blank line?

File: cli.js [hunk 2]
```diff
@@ -218,28 +220,30 @@ function processDirectory(inputDir, outputDir) {
       files.forEach(function(file) {
         var inputFile = path.join(inputDir, file);
         var outputFile = path.join(outputDir, file);
-        fs.readFile(inputFile, { encoding: 'utf8' }, function(err, data) {
-          if (!err) {
-            var minified;
-            try {
-              minified = minify(data, createOptions());
+        if (fileExt ? path.extname(file) === '.' + fileExt : true) {
```
Comment (alexlamsl) [inline cli.js:223]: I'd have gone with `Array.filter()`, but this also works.

**Comment by alexlamsl** [comment]
Thanks 😉

**Pull request merged**